Vagrantfile

```
@@ -0,0 +1,42 @@
+DO_BOX_URL = "https://github.com/devopsgroup-io/vagrant-digitalocean/raw/master/box/digital_ocean.box"
+PRIVATE_KEY_PATH = ENV["DIGITAL_OCEAN_PRIVATE_KEY_PATH"]
+TOKEN = ENV["DIGITAL_OCEAN_PAT"]
+
+Vagrant.configure("2") do |config|
+  config.ssh.insert_key = false
+  config.vm.define "droplet1" do |droplet|
+    droplet.vm.provider :digital_ocean do |provider, override|
+      override.ssh.private_key_path = PRIVATE_KEY_PATH
+      override.vm.box = 'digital_ocean'
+      override.vm.box_url = DO_BOX_URL
+      override.nfs.functional = false
+      override.vm.allowed_synced_folder_types = :rsync
+      provider.token = TOKEN
+      provider.image = 'ubuntu-22-04-x64'
+      provider.region = 'nyc1'
```

New york city er OK langt væk
```suggestion
      provider.region = 'fra1'
```